### PR TITLE
Diogoalmeida/ch3374/2 way tls for utimaco connecto rand tx logic

### PR DESCRIPTION
**Pull request by diogorac**
First https://github.com/RiddleAndCode/softhsm-mod/pull/19

### CODE REVIEW

File: cas/Makefile [hunk 1]
```diff
@@ -27,6 +28,7 @@ session.securosys.yml: mrenclave
 		| sed -e "s/\$$CONNECTOR_HOST/securosys_connector/g" \
 		| sed -e "s/\$$CONNECTOR_PORT/3009/g" \
 		| sed -e "s/\$$CONNECTOR_TYPE/SECUROSYS/g" \
+		| sed -e "s/\$$TLS2WAY/${call TLS2WAY}/g" \
```
Comment (jpopesculian) [inline cas/Makefile:31]: out of curiosity: what does call do here and why not just `${TLS2WAY}`?

**Review by jpopesculian — APPROVED** [review]
just a question

**Review by jpopesculian — APPROVED** [review]

**Comment by jpopesculian** [comment]
what was the bug @diogorac ?

**Comment by diogorac** [comment]
> what was the bug @diogorac ?

https://github.com/RiddleAndCode/softhsm-mod/pull/19/commits/f8ea5d23917363348edb3eb2d4ca71bacc924214

**Pull request merged**